test/statmodels.jl

```diff
@@ -36,5 +36,9 @@ x3   0.453058  0.72525 0.999172 0.5567
 @test sprint(showerror, ConvergenceException(10, 0.2, 0.1)) ==
     "failure to converge after 10 iterations. Last change (0.2) was greater than tolerance (0.1)."
 
+@test sprint(showerror, ConvergenceException(10, 0.2, 0.1, "Try changing maxIter.")) ==
+    "failure to converge after 10 iterations. Last change (0.2) was greater than tolerance (0.1). Try changing maxIter."
+
```

Comment:
A single break is enough.

Reply:
I don't understand what you mean.

Reply:
I mean better remove the second new empty line.

Reply:
removed.